index.html

```diff
@@ -49,6 +50,140 @@ <h2 class="connect"> <a href="#">Let's connect</a></h2>
           </ul>
         </div>
       </section>
+    </div>
+
+
+    <section class="works">
+      <article class="card card-1">
+        <img src="./images/tonic1.png" alt="Tonic" class="project-image">
+
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card card-2">
```

Comment:
- [ ] Kindly make the image and paragraph responsive for mobile devices.
![Captura de pantalla_2021-09-22_17-19-56](https://user-images.githubusercontent.com/53111464/134430893-d06e2726-db0a-4a76-85e5-f2b62a53d1ea.png)